### PR TITLE
RIO-55 feat(protocol): modify subscription to have push channel

### PR DESCRIPTION
**Pull request by HatemAlSum**

### CODE REVIEW

**Comment by ashihaby** [comment]
typo in the commit message

**Comment by ashihaby** [comment]
ok to test

File: really-core/src/main/scala/io/really/package.scala [hunk 1]
```diff
@@ -120,9 +120,13 @@ package io {
 
     object Request {
 
-      case class Subscribe(ctx: RequestContext, body: SubscriptionBody) extends Request with RoutableToSubscriptionManager
+      //case class Subscribe(ctx: RequestContext, body: SubscriptionBody) extends Request with RoutableToSubscriptionManager
 
-      case class Unsubscribe(ctx: RequestContext, body: UnsubscriptionBody) extends Request with RoutableToSubscriptionManager
+      //case class Unsubscribe(ctx: RequestContext, body: UnsubscriptionBody) extends Request with RoutableToSubscriptionManager
```
Comment (ashihaby) [inline really-core/src/main/scala/io/really/package.scala]: why you commented that?, you should remove this line

File: really-core/src/main/scala/io/really/package.scala [hunk 2]
```diff
@@ -120,9 +120,13 @@ package io {
 
     object Request {
 
-      case class Subscribe(ctx: RequestContext, body: SubscriptionBody) extends Request with RoutableToSubscriptionManager
+      //case class Subscribe(ctx: RequestContext, body: SubscriptionBody) extends Request with RoutableToSubscriptionManager
```
Comment (ashihaby) [inline really-core/src/main/scala/io/really/package.scala]: why you commented that?, you should remove this line

File: really-core/src/test/scala/io/really/protocol/ResponseWritesSpec.scala [hunk 3]
```diff
@@ -11,16 +12,10 @@ import org.joda.time.DateTime
 import org.scalatest.{ Matchers, FlatSpec }
 import play.api.libs.json.{ JsNull, JsString, Json }
 
-class ResponseWritesSpec extends FlatSpec with Matchers {
+class ResponseWritesSpec extends BaseActorSpec {
 
   import ProtocolFormats.ResponseWrites.resultWrites
 
-  val ctx = RequestContext(
```
Comment (ashihaby) [inline really-core/src/test/scala/io/really/protocol/ResponseWritesSpec.scala]: why you removed that?

File: really-core/src/main/scala/io/really/package.scala (hunk 1 above)
Comment (HatemAlSum) [inline really-core/src/main/scala/io/really/package.scala]: Removed

File: really-core/src/test/scala/io/really/protocol/ResponseWritesSpec.scala (hunk 3 above)
Comment (HatemAlSum) [inline really-core/src/test/scala/io/really/protocol/ResponseWritesSpec.scala]: I will use the one in BaseActorSpec

**Pull request merged**